pkg/minikube/download/binary.go

```diff
@@ -86,3 +87,32 @@ func Binary(binary, version, osName, archName, binaryURL string) (string, error)
 	}
 	return targetFilepath, nil
 }
+
+// CrictlBinary download the crictl tar archive to the cache folder and untar it
+func CrictlBinary(k8sversion string, crictlVersion string) (string, error) {
+	// first we check whether crictl exists
+	targetDir := localpath.MakeMiniPath("cache", "linux", runtime.GOARCH, k8sversion)
```

Comment:
We should call `Binary` for this func, it will handle the cache check and other things we're duplicating

Reply:
but Binary is designed for kubernetes binary, e.g. instead Binary it calls binaryWithChecksumURL which is not suitable to download crictl from github